sql/server/pom.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.stargate.api.sql</groupId>
+    <artifactId>sql-base</artifactId>
+    <version>0.0.16-SNAPSHOT</version>
+  </parent>
+  <groupId>io.stargate.api.sql.server</groupId>
+  <artifactId>stargate-avatica-server</artifactId>
```

Comment:
Nit: Maybe drop the "avatica" to avoid leaking implementation and go with `io.stargate.api.sql.server.sql-server` or just `io.stargate.api.sql.server`

Reply:
Yeah, naming is hard :)

Avatica is leaked invariably by us forcing the client to use the Avatica JDBC driver, so I figured we might as well make it explicit in the server artefact name.

In theory we could split the server into the "core" dealing purely with Calcite and one (or more) connectors, the first being Avatica. I just did not want to introduce too many modules in the first commit, plus is it not certain at this point whether we are going to support other wire protocols.

I wanted to use "sql-server" at some point, but did not want it to possibly be confused with the MS SqlServer.

I'll probably drop the "stargate" prefix anyway.

How about `io.stargate.api.sql:avatica-server`

Reply:
It has been suggested that one alternative to Avatica would be to implement the [PostgreSQL wire protocol](https://www.postgresql.org/docs/9.3/protocol.html). This is a bit more effort than Avatica server side, but it would advantages in term of number of languages supported (there is more postgresSQL clients than Avatica ones), and possibly in term of clients maturity.

I don't know how and if this impact this particular comment because we probably want to stick with Avatica a bit longer (or ever, really, not saying going the PostgreSQL wire protocol route is a given) for convenience as SQL is far from being production ready, and `avatica-server` is probably just fine here for now. But wanted to add that context.

Reply:
Right now, there's no clean line between Stargate and Avatica because Calcite itself has ties to Avatica in many places. For example, we have to produce values of certain types in result sets for Avatica to be able to serialize them correctly, even though we do not explicitly depend on Avatica at that level. It should certainly be possible to support multiple drivers, but at this stage I'm just not sure it is worth the effort.

Reply:
> at this stage I'm just not sure it is worth the effort

Everything's in the definition of "this stage" ;).

As a said, I agree we should ride on Avatica for a bit longer, because before worrying what exact client driver is better for users, we need to make the whole thing usable.

But the more I think about it, the more I think switching to the PostgreSQL wire protocol is something we should seriously consider (again, in due time), because it will give use substantially more clients. Including things like ODBC, which opens quite a few doors.

But again, not something for this PR. Just to keep in mind that if we see opportunities to avoid tying things too much to Avatica as we work on this moving on, then it's worth a bit of extra effort.

Reply:
By "this stage" I meant "this PR" :)

+1 to trying the PostgreSQL protocol.

Reply:
renamed artifact to `io.stargate.api.sql:avatica-server` 

and bundle to `Avatica-Server` with symbolic name `io.stargate.api.sql.server.avatica`